content/de-DE/learn/overview.smd

```diff
@@ -0,0 +1,512 @@
+---
+.title = "Übersicht",
+.author = "",
+.date = @date("2024-09-14:00:00:00"),
+.layout = "page.shtml",
+.custom = {
+	"mobile_menu_title": "Übersicht",
+},
+---
+# Funktionshighlights
+## Kleine, einfache Sprache
+
+Der Fokus liegt darauf die Anwendung zu debuggen, anstatt seine Kenntnisse der Programmiersprache zu debuggen.
+
+Die gesamte Syntax von Zig wird mit einer [500-zeiligen PEG-Grammatikdatei](https://ziglang.org/documentation/master/#Grammar) angegeben.
+
+Es gibt **keinen versteckten Kontrollfluss**, keine versteckten Speicherzuweisungen, keinen Präprozessor und keine Makros. Wenn es nicht so aussieht, als würde der Zig-Code wegspringen, um eine Funktion aufzurufen, dann tut er das auch nicht. Das bedeutet, dass man sicher sein kann, dass der folgende Code nur `foo()` und dann `bar()` aufruft, und das ist garantiert, ohne dass man die Typen von irgendetwas kennen müsste:
+
+```zig
+var a = b + c.d;
+foo();
+bar();
+```
+
+Beispiele versteckten Kontrollflusses:
+
+* D hat `@property`-Funktionen, das sind Methoden, die man mit etwas aufruft, das wie ein Feldzugriff aussieht, im obigen Beispiel könnte also `c.d` eine Funktion aufrufen.
+* C++, D und Rust haben Operatorüberladung, so dass der Operator „+“ eine Funktion aufrufen kann.
+* C++, D und Go haben 'throw/catch'-Ausnahmen, also könnte `foo()` eine Ausnahme auslösen und verhindern, dass `bar()` aufgerufen wird.
+
+Zig fördert die Code-Wartung und Lesbarkeit, indem der gesamte Kontrollfluss ausschließlich mit Sprachschlüsselwörtern und Funktionsaufrufen verwaltet wird.
+
+## Leistung und Sicherheit: Wähle zwei
+
+Zig hat vier [Build-Modi](https://ziglang.org/documentation/master/#Build-Mode), und sie können alle bis zur [Granularität des Geltungsbereichs](https://ziglang.org/documentation/master/#setRuntimeSafety) gemischt und angepasst werden.
+
+| Parameter | [Debug](/documentation/master/#Debug) | [ReleaseSafe](/documentation/master/#ReleaseSafe) | [ReleaseFast](/documentation/master/#ReleaseFast) | [ReleaseSmall](/documentation/master/#ReleaseSmall) |
+|-----------|-------|-------------|-------------|--------------|
+Optimierungen – verbessern die Geschwindigkeit, beeinträchtigen das Debugging, beeinträchtigen die Kompilierzeit | | -O3 | -O3| -Os |
+Laufzeitsicherheitsprüfungen – beeinträchtigen die Geschwindigkeit, beeinträchtigen die Größe, führen zu Abstürzen statt undefiniertem Verhalten | On | On | | |
```

Comment:
The longer text is messing with the table. It may be worth to change the css instead of the content if appropriate.

German:
![Screenshot from 2024-09-17 14-46-58](https://github.com/user-attachments/assets/ad45de3b-07aa-4fb1-b279-9329ad1c1dda)

English:
![Screenshot from 2024-09-17 14-47-39](https://github.com/user-attachments/assets/c8942767-9b89-4dd6-a7b5-29fe87e3b9b3)

Reply:
Yeah this is a problem but also all those doc pages do need to be revamped so I wouldn't worry *too* much about relatively minor things at the moment. I'm fine with whatever you settle on.